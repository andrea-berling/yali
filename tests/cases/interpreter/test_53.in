// The logical OR operator should return the first value that is truthy
if (false or "ok") print "baz";
if (nil or "ok") print "baz";
if (false or false) print "bar";
if (true or "bar") print "bar";
if (56 or "hello") print "hello";
if ("hello" or "hello") print "hello";
