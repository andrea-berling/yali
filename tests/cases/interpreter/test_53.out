baz
baz
bar
hello
hello
